core/SymbolRef.h

```diff
@@ -560,7 +560,7 @@ class Symbols {
     }
 
     static constexpr int MAX_SYNTHETIC_CLASS_SYMBOLS = 200;
-    static constexpr int MAX_SYNTHETIC_METHOD_SYMBOLS = 30;
+    static constexpr int MAX_SYNTHETIC_METHOD_SYMBOLS = 31;
```

Comment:
I don't remember seeing this need to be bumped frequently. What was the problem you ran into that caused this to be bumped?

Reply:
This check winds up failing, since we now register `<Magic>.<nil-for-safe-navigation>`:

https://github.com/sorbet/sorbet/blob/20fad5888ade5aa7a2605c3a1aba5ddf90b3bd90/core/GlobalState.cc#L734-L736

Reply:
gotcha